nuprojs/Microsoft.ServiceFabric.Actors/Microsoft.ServiceFabric.Actors.nuproj

```diff
@@ -94,7 +94,7 @@
       </File>
       <File Include="$(DropFolderNetFramework)\Microsoft.ServiceFabric.Actors.targets">
```

Comment:
You need to bump the nuget version in props file as well inorder for Windows Fabric repo to consume your changes.

Reply:
Bumped up to 5.0.4